src/router/products.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const db = require('../db');
+
+function allProducts(req, res, next) {
+  db.getAllProducts()
+    .then((products) => {
+      res.render('products', {
+        name: 'All Products',
+        products: products,
+      });
+      next();
+    });
+}
+
+module.exports = { allProducts: allProducts };
```

Comment:
You dont really need to specify allProducts twice if the same name. Only needed if alias naming is different from function name.